lib/pause.js

```diff
@@ -14,11 +14,12 @@ let rl;
 let nextStep;
 let finish;
 let next;
-let registeredVariables;
+let registeredVariables = {};
 /**
  * Pauses test execution and starts interactive shell
  */
-const pause = function (passedObject) {
+const pause = function (passedObject = {}) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must not be padded by blank lines.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4871039)